lib/sidekiq/cron/web_extension.rb

```diff
@@ -3,70 +3,112 @@ module Cron
     module WebExtension
 
       def self.registered(app)
-
         app.settings.locales << File.join(File.expand_path("..", __FILE__), "locales")
 
         #index page of cron jobs
         app.get '/cron' do
-          view_path    = File.join(File.expand_path("..", __FILE__), "views")
+          view_path = File.join(File.expand_path("..", __FILE__), "views")
+
+          @current_namespace = 'default'
+
+          @namespaces = Sidekiq::Cron::Namespace.all_with_count
 
+          # Not passing namespace takes all the jobs from the default one.
           @cron_jobs = Sidekiq::Cron::Job.all
 
           render(:erb, File.read(File.join(view_path, "cron.erb")))
         end
 
+        app.get '/cron/namespaces/:name' do
+          view_path = File.join(File.expand_path("..", __FILE__), "views")
+
+          @current_namespace = route_params[:name]
+
+          @namespaces = Sidekiq::Cron::Namespace.all_with_count
+
+          @cron_jobs = Sidekiq::Cron::Job.all(@current_namespace)
+
+          #if Slim renderer exists and sidekiq has layout.slim in views
+          if defined?(Slim) && File.exists?(File.join(settings.views,"layout.slim"))
+            render(:slim, File.read(File.join(view_path, "cron.slim")))
+          else
+            render(:erb, File.read(File.join(view_path, "cron.erb")))
+          end
```

Comment:
This block can be simplified as:

```suggestion
          render(:erb, File.read(File.join(view_path, "cron.erb")))
```